ner_v2/detectors/temporal/date/readme.md

```diff
@@ -0,0 +1,152 @@
+
+   1. ***relative_datedate***:  date which is derived with reference to current date.
+   2. ***day_literal***:  Words literally spoken for days like दिन,  दिवस.
+   3. ***month_literal***: Words literally spoken for months like महीना, मास, महीने.
+   4. **weekday**:  Words which represent weekday like सोमवार, मंगलवार.
+   5. ***month***: Words which represent month like जनवरी, फरवरी.
+   6. ***month_date_ref***: Words which present some day of month like 2 *तारिक*.
+
+   For reference see `date_constant.csv` file for hindi - https://github.com/hellohaptik/chatbot_ner/blob/language_datetime_code_review/ner_v2/detectors/temporal/date/hi/data/date_constant.csv
+
+   
+
+3. **datetime_diff_constant.csv**:  This file contains the vocabs which come in reference for both date and time entity. These vocabs help to reference to some datetime in past and future w.r.t current time or create diff in mentioned time. Examples are  **पिछले** दिन,  **पिछले** घंटे
+
+   |           key           | present_in_start | adding_magnitude |   datetime_type   |
+   | :---------------------: | :--------------: | :--------------: | :---------------: |
+   |       बाद \| baad       |      False       |        1         | add_diff_datetime |
+   |    इस \|  इसी \| isi    |       True       |        0         | add_diff_datetime |
+   | पिछले \| पिछला\| pichhla |       True       |        -1        | add_diff_datetime |
+   |  सावा \| sava \| sawa   |       True       |       0.25       |   ref_datetime    |
+   |      पौने \| paune       |       True       |      -0.25       |   ref_datetime    |
+
+   Here, 1st column contains the word which can in reference for date time, 2nd column define the positioning of this word in entity whether it comes in start or in end. For example, in sentence *2 din baad* ,  "baad" came after "2 din"(date entity), hence its present_in_start will be false.  3rd column define the magnitude it is adding to referenced date or time, (1 if its adding positively, -1 if adding negatively). 4th column define the datetime type. Below are their detailed description.
+
+   1. ***add_diff_datetime***:  Words which are referencing to datetime difference from current time like word baad in 2  दिन *बाद*  referring to a day, which comes 2 day later from current date.
+
+   2. **ref_datetime**: Words which referenced to create difference in time reference next to it. Example- सावा 4 , here whole sentence is referring to time 4:15.
+
+      
+
+      For reference see `datetime_constant.csv` file for hindi - https://github.com/hellohaptik/chatbot_ner/blob/language_datetime_code_review/ner_v2/detectors/temporal/date/hi/data/datetime_diff_constant.csv
```

Comment:
What is datetime_constant.csv? I can't find it in the repo. Remove this if not required.

Reply:
Also please put one example of adding a custom pattern. Add it for hindi if required.